src/components/ClayDropdown.vue

```diff
@@ -0,0 +1,336 @@
+<script setup lang="ts">
+    import { ref, computed } from "vue";
+
+    interface Props {
+        items?: { label: string; value: string }[];
+        placeholder?: string;
+        modelValue?: string;
+    }
+
+    const props = withDefaults(defineProps<Props>(), {
```

Comment:
Alcuni suggeriscono di usare il `defineProps` wrappato dal metodo `withDefault`.  
Io non sono d'accordo: mi sembra solo una "soluzione" ad un problema che - in realtà - non esiste.

In Vue le proprietà si possono definire in una sola chiamata, senza definire alcuna interfaccia extra, usando correttamente il metodo `defineProps`.  
Guarda negli altri file come funziona.